.github/workflows/main.yaml

```diff
@@ -678,6 +678,6 @@ jobs:
           files: |
             ${{ github.workspace }}/windows-*-x64*.zip
             ${{ github.workspace }}/windows-*-x86*.zip
-            ${{ github.workspace }}/macos-*-arm64.tar.xz
-            ${{ github.workspace }}/macos-*-x86_64.tar.xz
-            ${{ github.workspace }}/macos-*-universal.tar.xz
+            ${{ github.workspace }}/macos-*-arm64*.tar.xz
+            ${{ github.workspace }}/macos-*-x86_64*.tar.xz
+            ${{ github.workspace }}/macos-*-universal*.tar.xz
```

Comment:
I'd prefer a more exclusive glob expression here, e.g. `macos-*-arm64?(-dSYMs).tar.xz`.

The GitHub Actions runtime probably uses `node-glob` which explicitly supports Bash' `extglob` patterns.

Reply:
The `softprops/action-gh-release` uses the `glob` npm package, which does.

This is not a workflow glob, but an input to a Node-based action.

Reply:
Changed; made sure the glob expression worked out-of-band and it seems to.